app/views/friendships/index.html.haml

```diff
@@ -0,0 +1,37 @@
+%section.people
+  %header
+    %h2 People I Follow
+  %table.table
+    %thead
+      %tr
+        %th(width="30%")
+        %th(width="20%") Videos in Queue
+        %th(width="20%") Followers
+        %th(width="30%") Unfollow
+    %tbody
+      - current_user.following_relationships.each do |relationship|
+        %tr
+          %td
+            = link_to "#{relationship.leader.full_name}", user_path(relationship.leader)
```

Comment:
Instead of `user_path(relationship.leader)` you should use shorthand - `relationship.leader`.

You don't need to interpolate full_name.

`= link_to relationship.leader.full_name, relationship.leader`